rust/src/args.rs

```diff
@@ -36,6 +36,7 @@ use crate::debug::{d, Dt};
 /// options.version = true;
 /// options.debug = true;
 /// options.debug_level = 3;
+/// options.raw_mode = true;
```

Comment:
Nit: I'd love the keep the debug flags at the bottom and version right above them?
I'd even move it further up to right below `env`. Seems appropriate?

Reply:
Sure! I don't mind nit-picking at all. It's your project, after all :)